app/Http/Requests/Devices/DeviceUpdateRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Requests\Device;
+
+use App\Http\Requests\Request;
+
+class DeviceUpdateRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
zet dit anders ff in de comments, dan is het wat logischer voor de casual observer